Mathlib/RingTheory/Bialgebra/GroupLike.lean

```diff
@@ -0,0 +1,167 @@
+/-
+Copyright (c) 2025 Yaël Dillies, Michał Mrugała. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Michał Mrugała
+-/
+import Mathlib.RingTheory.Bialgebra.Equiv
+import Mathlib.RingTheory.Flat.Domain
+
+/-!
+# Group-like elements in a bialgebra
+
+This file defines group-like elements in a bialgebra, ie units `a` such that `Δ a = a ⊗ₜ a`.
+
+The motivating example is the group ring `R[G]`. Group-like elements of `R[G]` are exactly the image
+of `G` inside `R[G]`. We note two things: these group-like elements form a group and are linearly
```

Comment:
> Group-like elements of `R[G]` are exactly the image
of `G` inside `R[G]`

I think this is false over arbitrary rings according to Sophie Morel?

Reply:
Yes, sorry